include/config_file.h

```diff
@@ -32,6 +32,15 @@ typedef enum {
 	R_CONFIG_SYS_VARIANT_NAME,
 } RConfigSysVariant;
 
+typedef struct {
+	gboolean boot_id;
+	gboolean transaction_id;
+	gboolean machine_id;
+	gboolean serial;
+	gboolean variant;
+	gboolean uptime;
+} RConfigHTTPSendHeaders;
+
```

Comment:
The idea was to be more performant with integer comparison rather than requiring string comparison. Maybe it's not that relevant here. I have changed it to use string lists..